runelite-client/src/main/java/net/runelite/client/plugins/fairyring/FairyRingPlugin.java

```diff
@@ -68,6 +76,8 @@
 )
 public class FairyRingPlugin extends Plugin
 {
+	public static final String CONFIG_GROUP = "fairyrings";
```

Comment:
We usually define this in the config interface, but it's not really too big a deal where it ends up.